examples/simple/simple.cpp

```diff
@@ -22,6 +60,10 @@ int main(int argc, char ** argv) {
         params.prompt = argv[2];
     }
 
+    if (argc >= 4) {
+        observe = atoi(argv[3]);
```

Comment:
I am not sure if this causes any issues in practice, but a common idiom to convert int to bool is to use `!!`.
```suggestion
        observe = !!atoi(argv[3]);
```